src/Dapr.PluggableComponents.Protos/Dapr.PluggableComponents.Protos.csproj

```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <ProtosVersion>v1</ProtosVersion>
-    <ProtosTag>v1.11.0</ProtosTag>
+    <ProtosTag>master</ProtosTag>

```

Comment:
We'll want to defer committing this PR until (at least) a `v1.12.0` tag exists in `dapr/dapr`, as we don't want builds picking up whatever happens to be in the `master` branch at the time.